packages/genesys-spark-components/src/components/stable/gux-modal/gux-modal.tsx

```diff
@@ -84,7 +84,7 @@ export class GuxModal {
     switch (event.key) {
       case 'Escape':
         event.preventDefault();
-        this.dialogElement.close();
+        this.open = false;
```

Comment:
This is a nitpick but could you just call `this.onDismissHandler()` here instead as it does the same thing ?